Docusaurus: add intergrations

StrapiShaun opened the pull request:
This PR converts the integrations to Docusaurus. 
- There is a small update to the introductions.
- Conclusion sections have been removed.
- v4 caution callout is removed.
- `Request/Response`, `ApiCall`, and `Tabs` have been formatted for Docusaurus. 
- Obvious spelling and grammar mistakes are fixed.

**Guides are not technically validated or rewritten**


- [x] guides introduction
- [x] react
- [x] vue
- [x] angular
- [x] next
- [x] nuxt
- [x] graphql
- [x] gatsby
- [x] girdsome
- [x] jekyll
- [x] 11ty
- [x] svelte
- [x] sapper
- [x] ruby
- [x] python
- [x] dart
- [x] flutter
- [x] go
- [x] php
- [x] Laravel
- [x] update TOC
- [x] update status.md

StrapiShaun commented:
@pwizla One more thing. I can improve the cards on the intro page before we merge. They are pulling the description which doesn't fit on the card. Let me know if you have an opinion on the best way to address this.

pwizla reviewed (changes requested):
Thanks, Shaun! This is a huge PR, great job!

For the cards on the intro page, what do you think about "Integrate Strapi with [technology name]"? I think it would fit in. If we have enough room, it could be "Query content from Strapi with [technology name]" (because this is basically what our guides do, once the tech. is installed) but I'm not sure it will fit.

Apart from that, it's almost ready :) 
I would maybe just suggest that we use the `noSideBySide` argument on API calls to that response blocks are always below queries, otherwise, I find it a bit harder to read. You did it, I think, on some pages, but I noticed for instance that the Dart, Gridsome, GO, Python, Ruby, and GraphQL guides could benefit from it :)

We also have a few rendering issues with the PHP guide (see example screenshot)
![Screenshot 2023-01-24 at 18 18 53](https://user-images.githubusercontent.com/4233866/214362704-9e154b6a-68db-4f7f-9a73-cf5e4cfd8440.png)

StrapiShaun commented:
> Thanks, Shaun! This is a huge PR, great job!
> 
> For the cards on the intro page, what do you think about "Integrate Strapi with [technology name]"? I think it would fit in. If we have enough room, it could be "Query content from Strapi with [technology name]" (because this is basically what our guides do, once the tech. is installed) but I'm not sure it will fit.
> 
> Apart from that, it's almost ready :) I would maybe just suggest that we use the `noSideBySide` argument on API calls (see [internal docs](https://www.notion.so/strapi/WIP-Docusaurus-writing-guide-b26679ef4839473e8948614453b11667#cb83ea2ab1c0443cae699c47aad12afc)) to that response blocks are always below queries, otherwise, I find it a bit harder to read. You did it, I think, on some pages, but I noticed for instance that the Dart, Gridsome, GO, Python, Ruby, and GraphQL guides could benefit from it :)
> 
> We also have a few rendering issues with the PHP guide (see example screenshot) ![Screenshot 2023-01-24 at 18 18 53](https://user-images.githubusercontent.com/4233866/214362704-9e154b6a-68db-4f7f-9a73-cf5e4cfd8440.png)

Small typo for `noSideBySide`. Fixed in all of the affected guides. I think this also fixed the issue with the PHP guide.

pwizla reviewed (approved):
Thanks, Shaun! LGTM now. 
For the `description` tag both solutions work for me. You can merge the PR once we settled on any of them and updated the tags content accordingly (maybe ask the marketing as well? 🤔)

StrapiShaun commented:
I checked with marketing and settled on "integrate".

StrapiShaun commented:
Yes, I just fixed this directly on the `next` branch. Thanks!

On Wed, Jan 25, 2023 at 10:50 AM Gabriel ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docusaurus/docs/dev-docs/integrations/angular.md
> <https://github.com/strapi/documentation/pull/1356#discussion_r1086482837>
> :
>
> > +  </div>
> +
> +  <div *ngIf="allCategories">
> +    <br />
> +    Select categories
> +    <div *ngFor="let category of allCategories">
> +      <label>{{ category.name }}</label>
> +      <input [value]="category.id" type="checkbox" (change)="onCheckChange($event)" />
> +    </div>
> +  </div>
> +
> +  <button class="button" type="submit">Create</button>
> +</form>
> +```
> +
> +### PUT Request your collection type
>
> Should this be H2?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/strapi/documentation/pull/1356#pullrequestreview-1269111872>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEIOALMMS7NSIXHVV5ZND3WUEAQ5ANCNFSM6AAAAAATCOGMVQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>


-- 

Shaun Brown

Technical Writer


Website <https://strapi.io/> | Get a demo <https://strapi.io/demo> | Roadmap
<https://feedback.strapi.io/>  | We're hiring! <https://strapi.io/careers>
